src/quacc/recipes/newtonnet/ts.py

```diff
@@ -264,6 +267,246 @@ def quasi_irc_job(
     return relax_summary
 
 
+@job
+@requires(
+    has_newtonnet, "NewtonNet must be installed. Refer to the quacc documentation."
+)
+@requires(
+    has_geodesic_interpolate,
+    "geodesic-interpolate must be installed. Refer to the quacc documentation.",
+)
+def neb_job(
+    reactant_atoms: Atoms,
+    product_atoms: Atoms,
+    relax_job_kwargs: dict[str, Any] | None = None,
+    calc_kwargs: dict[str, Any] | None = None,
+    geodesic_interpolate_kwargs: dict[str, Any] | None = None,
+    neb_kwargs: dict[str, Any] | None = None,
+) -> dict:
+    """
+    Perform a quasi-IRC job using the given atoms object. The initial IRC job by default
+    is run with `max_steps: 5`.
+
+    Parameters
+    ----------
+    atoms
+        The atoms object representing the system
+    direction
+        The direction of the IRC calculation
+    relax_job_kwargs
+        Keyword arguments to use for the [quacc.recipes.newtonnet.core.relax_job][]
+
+    Returns
+    -------
+    A dictionary containing the neb optimization summary.
+            See the type-hint for the data structure.
+    """
```

Comment:
This is not how a `Returns` is formatted. You need to have it like

```python
dict
    Description goes here
```

otherwise the docs do not build. This is called numpy docstring format.